build.gradle

```diff
@@ -189,7 +190,10 @@ dependencies {
 	implementation 'org.webjars:bootstrap:3.3.7'
 	implementation 'org.webjars:jquery:3.1.1-1'
 
-	//Hibernate
+	//JSONtoXML
+	implementation 'org.json:json:20211205'
```

Comment:
What do you use that for ?

Reply:
We use it in EagreementServiceImpl.kt to go back and forth between XML and JSON objects